hrm-domain/hrm-core/profile/profileDataAccess.ts

```diff
@@ -13,8 +13,6 @@
  * You should have received a copy of the GNU Affero General Public License
```

Comment:
Why removing all `Result`s from the db layer? Isn't here where we get the most benefit from having it? I imagined that if we wrap all DB calls in Results we wont need to try/catch at service layers anymore.

Reply:
If there isn't specific logic handling the errors from a data access method by the calling code, the result type is adding any value so is removed. 

If we the service layer cares about the error types coming from here, we can use a result type, otherwise we can lose the cruft. This is aligned with what we discussed about using result types in a more limited way, IIRC?

Reply:
I thought we would wait for the rest of the team to get a saying on how/where we want to use this, but is fine either way tbh. Just asking to know if the big refactor was actually needed for the rest of the changes.